doc/lightning-makesecret.7.md

```diff
@@ -35,7 +36,7 @@ Aditya <<aditya.sharma20111@gmail.com>> is mainly responsible.
 SEE ALSO
 --------
 
-lightning-getsharedsecret(7)
+lightning-datastore(7)
```

Comment:
Why is this referenced here?

Reply:
Hmm, mainly because it uses a similar API.  It *is* weird though, I'll remove it...